src/pages/admin-ui-sdk/reference/menu-app.md

```diff
@@ -1,6 +1,62 @@
 ---
 title: MenuApp component
-description: 
+description: MenuApp component props and usage.
 ---
 
 # MenuApp component
+
+`MenuApp` adds a menu item to the Commerce Admin UI menu page. This component selects the DOM element on the Admin menu page that matches its `rootElmSelector` prop. This is where the `MenuApp` renders a new menu item with a URL to the out-of-process App Builder page used to add Commerce features.
+
+## MenuApp Props
+
+| Name              | Type     | Required | Default | Description                                 |
+| ----------------- | -------- | -------- | ------- | ------------------------------------------- |
+| `menuHtmlUrl`     | `string` | true     | null    | URL to the out-of-process App Builder page. |
```

Comment:
The `menuHtlmUrl` is the URL to retrieve the Commerce Menu to append it with the out-of-process App Builder registrations.